<launch>

  <node pkg="cob_datamatrix" type="datamatrix.py" name="datamatrix" output="screen">
	<remap from="image_rgb" to="/cam3d/rgb/image_color"/>
	<remap from="camera_info" to="/cam3d/rgb/camera_info"/>
	<remap from="pointcloud_depth" to="/cam3d/depth/points"/>
  </node>

</launch>
